src/main/js/custom_commands/removePageHead.js

```diff
@@ -1,13 +1,13 @@
-/**
- * Nightwatch command to remove the breadcrumb bar on classic jenkins pages.
+/** @module removePageHead
```

Comment:
VERY VERY MINOR NIT: comments starting on the first line and inconsistent spacing with the other like this